packages/studio-plugin/src/viteconfig/getStudioViteOptions.ts

```diff
@@ -11,6 +11,7 @@ export default function getStudioViteOptions(
     open:
       args.mode === "development" &&
       args.command === "serve" &&
+      !process.env.YEXT_CBD_BRANCH &&
```

Comment:
would it be better to check that `xdg-open` exists instead of tying the check to CBD details?

Reply:
I can't find any way to check if it exists